random.c

```diff
@@ -0,0 +1,29 @@
+/*Matthew Danielson
+* 6/29/16
+* Random.c
+*Generates a random number to be generated
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main(){
+	srand(time(NULL));
+	int randint = rand();
+	int randint2 = randint%10 ;
+	randint2 ++;
+	int num = 1;
+	int guess;
+	while(num){
```

Comment:
I think because of the way you implemented this while loop, you have to insert two guesses before your program works.